wandb/internal/stats.py

```diff
@@ -81,7 +81,8 @@ def start(self):
             self._shutdown = False
             self._thread = threading.Thread(target=self._thread_body)
             self._thread.daemon = True
-        self._thread.start()
+        if not self._thread.is_alive():
+            self._thread.start()
```

Comment:
This fixes the issue @cvphelps mentioned in #cling, in notebooks we send a resume request on cell start.  If you call `wandb.init` in a different cell and the pause didn't happen somehow, this can get triggered.